Justfile

```diff
@@ -339,4 +339,9 @@ clean-slate-liftoff: clean-sdk
 	just publish-local-compiler-plugin
 	just publish-local-provider-sdk kubernetes 4.2.0
 	just clean-liftoff
-	just liftoff
\ No newline at end of file
+	just liftoff
+
+# Runs 'scala-cli setup-ide' for all modules
+mass-setup-ide:
+	for file in `ls */project.scala | cut -f1 -d'/'`; do scala-cli setup-ide $file; done
+	for file in `ls */*/project.scala | cut -f1,2 -d'/'`; do scala-cli setup-ide $file; done
```

Comment:
Hmmm, not sure if this is the way to do it. Does this work for you in metals?
I was convinced that we have to use `scala-compose` and it takes an additional parameter. Honestly, I'm not sure exactly how it should work. The command I used was e.g. `scomp setup-ide --conf-dir . core` (where `scomp` is my alias to the `scala-compose` binary)

Reply:
this works with IntelliJ :D